### PR TITLE
feat(didSync): add concurrency limit to refresh processing

### PR DESCRIPTION
**Pull request by jrhender**
I don't have any great evidence for it but I think it is possible that DGraph locking up could be the cause of the iam-cache-server going down. To reduce what might be a cause of strain on DGraph, I am thinking that it might be a good idea to limit the number of DidDoc refreshes that occur concurrently so that DGraph (and the blockchain RPC) it only given a new request once it is done processing a previous one. Do you think this is a good idea?

### CODE REVIEW

**Review by dwojno — APPROVED** [review]

**Comment by jrhender** [comment]
Will finalize and merge this once the auth changes have been merged.

**Comment by Harasz** [comment]
@jrhender What is the status of this PR?

**Comment by Harasz** [comment]
I am closing the thread due to inactivity.

**Pull request closed without merging**
